drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -8,7 +8,7 @@ function DragAndDropTemplates(configuration) {
         }
         return (
             h("div.spinner-wrapper", {key: item.value + '-spinner'}, [
-                h("i.fa.fa-spin.fa-spinner")
+                h("span.fa.fa-spin.fa-spinner")
```

Comment:
Need also to add aria-hidden (and make sure there is visible or sr text). See markup at http://ux.edx.org/design_elements/icons/.